src/replayToMovie.js

```diff
@@ -13,7 +13,7 @@ const CRF = process.env.QUALITY || 23;
 // Allow binaries to run out of the bundle
 process.env['PATH'] += ':' + process.env['LAMBDA_TASK_ROOT'];
 
-const SPRITE_S3_BASE = "http://s3.amazonaws.com/cdo-curriculum/images/sprites/spritesheet_tp/";
+const SPRITE_S3_BASE = "http://s3.amazonaws.com/cdo-curriculum/images/sprites/spritesheet_tp2/";
```

Comment:
Note: `dance-party` [is now using](https://github.com/code-dot-org/dance-party/blob/master/src/ResourceLoader.js#L1) a base asset path `/images/sprites/dance_grouped/` rather than `/images/sprites/spritesheet_tp2`.  `dance_grouped` has the change that packs _all_ animation json into one file, and only publishes four pngs per character.

Reply:
Let me see if it's straightforward to update this to use that process

Reply:
If it's not, I think this is fine too - both sets of assets were generated from the same source images and should give the same results.

Reply:
it's not straightforward enough that it's something I want to do now; I'll add an issue for tracking it as a longer-term work item